tests/test_read.py

```diff
@@ -0,0 +1,70 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import random
+import string
+from unittest import mock
+import pytest
+
+from azure.core.credentials import AzureSasCredential, AzureNamedKeyCredential
+from azure.identity import DefaultAzureCredential
+
+from azstoragetorch.io import BlobIO
+from azstoragetorch._client import AzStorageTorchBlobClient
+
+from azure.core.exceptions import ResourceExistsError
+from azure.storage.blob import BlobServiceClient
+
+
+
+@pytest.fixture
+def account_url():
+    account_name = os.environ.get('PYTORCH_STORAGE_ACCOUNT_NAME', None) or "pytorchstorageaccount"
+    return "https://" + account_name.strip('"') + ".blob.core.windows.net"
+
+@pytest.fixture
+def container_name():
+    return ''.join(random.choices(string.ascii_lowercase + string.digits, k=8))
+
+@pytest.fixture
+def blob_name():
+    return ''.join(random.choices(string.ascii_lowercase + string.digits, k=8))
+
+@pytest.fixture
+def sample_data():
+    return ''.join(random.choices(string.ascii_lowercase + string.digits, k=8)).encode()
+
+@pytest.fixture
+def container(
+    account_url, container_name
+):
+    blob_service_client = BlobServiceClient(account_url, credential=DefaultAzureCredential())
+
+    cont = blob_service_client.create_container(name=container_name)
+    yield blob_service_client
```

Comment:
For this fixture, lets:
1. Yield the container client back as part of the fixture. In general, the service client will likely not be needed for much past creating containers. So that will reduce the number of hops needed in downstream fixtures between client calls.
1. Rename it to `container_client`. This helps the fixture be a bit more self-documenting in that it is returning a container client as it could be interpreted that `container` could be a container name or url.